theano/tensor/tests/test_basic.py

```diff
@@ -1930,6 +1930,46 @@ def _numpy_true_div(x, y):
                         bad_build=dict(),
                         bad_runtime=dict(bad1=(rand(5, 7), rand(5, 7)),
                                          bad2=(rand(5, 7), rand(8, 3))))
+BatchedDotTester = makeTester(
```

Comment:
This being said, there seems to be insufficient test cases with involving tensors with less or more than 3 dimensions. Ex :  what happens if you call batched_dot with two vectors? two matrices? a scalar and something else? a 4d/5d tensor with something else?